backend/apid/routers/namespaces.go

```diff
@@ -30,8 +48,72 @@ func (r *NamespacesRouter) Mount(parent *mux.Router) {
 	}
 
 	routes.Del(r.handlers.DeleteResource)
-	routes.Get(r.handlers.GetResource)
+	routes.Get(r.get)
```

Comment:
The `r.get` method actually lists namespaces

Reply:
I realized I actually completely broke the router... oops.

I've fixed it now and reinstated the get unit tests.